multimedia/fluidsynth/files/patch-external_project_cc.diff

```diff
@@ -0,0 +1,10 @@
+--- src/CMakeLists.txt.orig	2018-12-30 04:42:00.000000000 -0700
++++ src/CMakeLists.txt	2019-02-05 05:02:59.000000000 -0700
+@@ -366,6 +366,7 @@
+ ExternalProject_Add(gentables
+     DOWNLOAD_COMMAND ""
+     SOURCE_DIR ${CMAKE_CURRENT_SOURCE_DIR}/gentables
++    CMAKE_ARGS "-DCMAKE_C_COMPILER=${CMAKE_C_COMPILER}"
```

Comment:
Setting CMAKE_BUILD_TYPE here is at least as important as setting the compiler. We use a custom build type to make CMake use the CFLAGS & family env. variables which I can more easily imagine as more important than the actual compiler. The generated tables could be different for 32bit vs. 64bit builds for instance.